elements/guzman-neilan.def

```diff
@@ -6,32 +6,33 @@ short-names:
   - GN
 ndofs:
   triangle:
-    formula: 11
+    formula: 9
   tetrahedron:
     formula:
-    - k=1: 19
-    - k=2: 49
+    - k=1: 16
+    - k=2: 34
```

Comment:
There's meant to be a test that checks that these values match the values from Symfem. I guess these being wrong means there may be an error in that test